File: src/DZarsky.TS3Viewer2.Domain/Server/Dto/ServerInfoDto.cs
namespace DZarsky.TS3Viewer2.Domain.Server.Dto;

public class ServerInfoDto
{
    public int Id { get; set; }
    
    public string? Status { get; set; }
    
    public int ClientsOnline { get; set; }
    
    public int QueriesOnline { get; set; }
    
    public int MaxClients { get; set; }
    
    public string? Uptime { get; set; }
    
    public string? Name { get; set; }
}

File: src/DZarsky.TS3Viewer2.Web/src/api/models/ServerInfoDto.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

export type ServerInfoDto = {
    id?: number;
    status?: string | null;
    clientsOnline?: number;
    queriesOnline?: number;
    maxClients?: number;
    uptime?: string | null;
    name?: string | null;
};